README.md

```diff
@@ -10,7 +10,279 @@ This capsule is the the companion code to the Bixby [Space Resorts Sample Capsul
 
 ---
 
-For use cases and annotation patterns, please see [annotations](./ANNOTATIONS.md)
+## Use cases
+
+### Finding Space Resorts
+
+#### Outer "find" Queries
+
+![Outer "find" Query](./assets/images/OuterFindQuery.png)
+
+You can see all trained utterances and plans by entering this query in the
+training tab search bar: `goal:SpaceResort#all -has:continue`. Examples:
+
+- Find space resorts
+
+- Show me space hotels with crater canyoneering
+
+- Search for hotels around The Red Planet
+
+- Look for space hotels with quantum bungee jumping around Saturn
+
+We train these to have the goal `SpaceResort#all` and we annotate any resort
+names, planets and search criteria as Values. Here `SpaceResort#all` is a
+property projection to the `all` property of the `SpaceResort`. This property is
+a boolean of type `ViewAll` which is always true and is used as a proxy to
+signify that we want to display the full space resort, instead of focussing on a
+single property like `gravity` or `planet` (see property projections below). We
+use a match-pattern to then tie our views and dialogs for this property
+projection to describe the whole hotel.
+
+```
+match {
+  ViewAll(all) {
+    from-property {
+      SpaceResort (result)
+    }
+  }
+}
+```
+
+Why do we treat this as a property projection instead of setting the goal to the
+`SpaceResort` Structure?
+
+When the user wants to book a space resort and there were many possible
```

Comment:
The tense in this sentence should be more consistent.

When the user wants to book a space resort and there are many possible candidates in context, the user picks a single one using a selection prompt, in order to continue with the booking (see SpaceResort Selection Prompt selection below for full details).